### PR TITLE
Added the General Purpose Service Robot task.

### PR DESCRIPTION
**Pull request by justinhart**
Help Out is intended to replace the General Purpose Service Robot task.

### CODE REVIEW

File: tasks/HelpOut.tex [hunk 1]
```diff
@@ -0,0 +1,86 @@
+\section{Help Out [Housekeeper]}
+Smart speakers have become a staple device in modern households only a few years after their introduction because they fill a niche for a general assistive device in the home. A clear goal of upcoming domestic service robot technologies is to have a device that aids users in day-to-day tasks.
```
[the remark this answers is not in the document]
Comment (johaq) [reply tasks/HelpOut.tex]: I think no other task has a justification for why it is useful. Should be self evident. If anything this sounds like something for the general rules but I would remove it here.

File: tasks/HelpOut.tex [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+% \subsection{Focus}
+% This test focuses on the detection and recognition of objects and their features, as well as object manipulation.
+
+\subsection{Main Goal}
+Accept verbal instructions from home occupants and perform those tasks.
```
[the remark this answers is not in the document]
Comment (johaq) [reply tasks/HelpOut.tex]: Maybe change accept to understand?

File: tasks/HelpOut.tex [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+\noindent\textbf{Reward:} 600pts (200 points per task)\\
+
+\subsection{Bonus rewards}
+\begin{enumerate}[nosep]
+	\item Free-form English commands. The team may ask the commands to be re-worded from the generated command, in order to test more advanced natural language processing. (100 pts, each)
+	\item Going home when instructed to do so. The robot must park itself in the home position and successfully come back when requested with, \textit {<Robot Name>, I need help!}. (50pts)
```
[the remark this answers is not in the document]
Comment (johaq) [reply tasks/HelpOut.tex]: Is driving to a designated position really bonus worthy?

File: tasks/HelpOut.tex [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+	\item Coming when called. Robot should come to the same room as the operator when requested with, \textit {<Robot Name>, I need help!}. (50pts)
```
[the remark this answers is not in the document]
Comment (johaq) [reply tasks/HelpOut.tex]: If the robot really only finds the way by sound I think this is really cool (especially in @home since there are no real walls...) and it would deserve more points.

File: tasks/HelpOut.tex (hunk 2 above)
[the remark this answers is not in the document]
Comment (justinhart) [reply tasks/HelpOut.tex]: Done!

File: tasks/HelpOut.tex (hunk 3 above)
[the remark this answers is not in the document]
Comment (justinhart) [reply tasks/HelpOut.tex]: Removed, and beefed up the bonus for coming to the operator in another room!

File: tasks/HelpOut.tex (hunk 4 above)
[the remark this answers is not in the document]
Comment (justinhart) [reply tasks/HelpOut.tex]: I beefed up the bonus for this, and removed the weaker bonus.

File: tasks/HelpOut.tex [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+% This test focuses on the detection and recognition of objects and their features, as well as object manipulation.
+
+\subsection{Main Goal}
+Accept verbal instructions from home occupants and perform those tasks.
+
+\noindent\textbf{Reward:} 600pts (200 points per task)\\
+
+\subsection{Bonus rewards}
+\begin{enumerate}[nosep]
+	\item Free-form English commands. The team may ask the commands to be re-worded from the generated command, in order to test more advanced natural language processing. (100 pts, each)
```
[the remark this answers is not in the document]
Comment (justinhart) [reply tasks/HelpOut.tex]: Beefed up to 150 pts.

File: tasks/HelpOut.tex [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+	\item \textbf{Object and Resident Placements:} Residents will instruct the robot to perform tasks in the various rooms of the home. Tasks should be pre-generated and the residents and items should be in the correct places before the round starts.
+\end{enumerate}
+
+
+% %% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%
+% Setup
+%
+% %% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\subsection{Procedure}
```
[the remark this answers is not in the document]
Comment (justinhart) [reply tasks/HelpOut.tex]: Removed!

File: tasks/HelpOut.tex [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+
+	\item \textbf{House Residents:} Three people in the arena will give tasks to the robot.
+
+	\item \textbf{Object and Resident Placements:} Residents will instruct the robot to perform tasks in the various rooms of the home. Tasks should be pre-generated and the residents and items should be in the correct places before the round starts.
+\end{enumerate}
+
+
+% %% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%
+% Setup
```
[the remark this answers is not in the document]
Comment (justinhart) [reply tasks/HelpOut.tex]: Fixed!

File: tasks/HelpOut.tex (hunk 1 above)
[the remark this answers is not in the document]
Comment (justinhart) [reply tasks/HelpOut.tex]: Changed.

**Comment by justinhart** [comment]
I think that I've addressed everything.

File: tasks/GPSR.tex [hunk 8]
```diff
@@ -1,19 +1,18 @@
...
 
-\noindent\textbf{Reward:} 600pts (200 points per task)\\
+\noindent\textbf{Reward:} 2500pts (325 points per task)\\
 
 \subsection{Bonus rewards}
 \begin{enumerate}[nosep]
-	\item Free-form English commands. The team may ask the commands to be re-worded from the generated command, in order to test more advanced natural language processing. (100 pts, each)
-	\item Going home when instructed to do so. The robot must park itself in the home position and successfully come back when requested with, \textit {<Robot Name>, I need help!}. (50pts)
-	\item Coming when called. Robot should come to the same room as the operator when requested with, \textit {<Robot Name>, I need help!}. (50pts)
+	\item Free-form English commands. The team may ask the commands to be re-worded from the generated command, in order to test more advanced natural language processing. (150 pts, each)
```
Comment (johaq) [inline tasks/GPSR.tex]: For clarification. Does the team recieve the bonus points just for understanding the free form command or do they only get the bonus if the also complete said command?

File: scoresheets/GPSR.tex [hunk 9]
```diff
@@ -1,71 +1,16 @@
...
+	\scoreitem[5]{325}{Perform each task}
 
-	\scoreheading{Leave the arena}
-	\scoreitem{10}{Leave the arena after successfully accomplishing a command}
+	\scoreheading{Bonus rewards}
+	\scoreitem[5]{150}{Respond to free-form English commands}
+	\scoreitem{125}{Coming when called}
 
-	\setTotalScore{250}
+	%\setTotalScore{1000}
```
Comment (kyordhel) [inline scoresheets/GPSR.tex]: 5 x 325 + 5 x 150 + 125 = 1625 + 600 + 125 = 2500

OK

File: tasks/GPSR.tex [hunk 10]
```diff
@@ -1,19 +1,18 @@
...
-Smart speakers have become a staple device in modern households only a few years after their introduction because they fill a niche for a general assistive device in the home. A clear goal of upcoming domestic service robot technologies is to have a device that aids users in day-to-day tasks.
+\section{General Purpose Service Robot [Housekeeper]}
+Similar to a modern smart-speaker, the robot can be asked to do anything, in plain English, from this rulebook or any previous rulebook.
 
 % \subsection{Focus}
 % This test focuses on the detection and recognition of objects and their features, as well as object manipulation.
 
 \subsection{Main Goal}
-Accept verbal instructions from home occupants and perform those tasks.
+Understand and act on verbal instructions from home occupants and perform those tasks.
```
Comment (kyordhel) [inline tasks/GPSR.tex]: Change home occupants to **operator**

File: tasks/GPSR.tex (hunk 8 above)
Comment (kyordhel) [reply tasks/GPSR.tex]: As is, only if the command is successfully executed.
